platform/core.multitabs/src/org/netbeans/core/multitabs/impl/DocumentSwitcherTable.java

```diff
@@ -174,7 +177,27 @@ public String getToolTipText( MouseEvent event ) {
         }
         return null;
     }
-
+    
+    boolean closeSelectedDocumentList() {
```

Comment:
Would be good to add Javadoc explaining what the return value is supposed to mean here.